### ISSUE TITLE
Next release?

### ISSUE THREAD

**Issue by davisyoshida** [jax-ml/jax-triton#127]
The current version of the pip package is incompatible with jax>0.4.5 due to the removal of the `ShapedArrayRef` type from `jax._src.state.types`. Will the pip package be updated soon to allow compatibility with the latest version of jax again?

**Comment by sharadmv** [comment]
Yes working on a release, will hopefully have it out in a few days

**Comment by sharadmv** [comment]
Okay I think this may be blocked for a bit. We depend on Triton close to HEAD and there aren't plans for a Triton release that soon. Once Triton is released, we can cut a release.

In the meantime, you can try to manually install jax-triton w/ a newer Triton and a newer JAX.

**Comment by davisyoshida** [comment]
I had some trouble getting bazel to find my CUDA installation, so I'm just using the version of jax-triton on pip and manually patching it to fix the references to ShapedArrayRef.

**Comment by sharadmv** [comment]
FYI I've updated the installation guide. You can now use CUDA installed via pip!

**Comment by davisyoshida** [comment]
Thanks! It looks like I can install it from source now, although I had to lower my gcc version to 10 to get triton to build. It looks like the add and softmax examples work, but the others crash for various reasons (e.g. I think fused_attention crashes due to https://github.com/openai/triton/issues/1572).
